packages/server/src/automations/steps/sendSmtpEmail.ts

```diff
@@ -72,10 +72,9 @@ export const definition: AutomationStepSchema = {
           title: "Location",
           dependsOn: "addInvite",
         },
-        url: {
-          type: AutomationIOType.STRING,
-          title: "URL",
-          dependsOn: "addInvite",
+        attachments: {
+          type: AutomationIOType.ATTACHMENT,
```

Comment:
Single attachment is not yet there, probably I should prioritise it in order to all work to consider this type

Reply:
Yeah - I'm wondering if the type is still needed to tell the automation UI that it needs to support multiple attachments rather than just a single one (key value builder vs a single key value)